app.py

```diff
@@ -137,9 +140,10 @@ def package_releases_update(monitored_packages: dict,
 
         if added:
             _LOGGER.info("Package %r in version %r was newly added", package_name, package_version)
-            _METRIC_PACKAGES_NEW.inc()
+            _METRIC_PACKAGES_NEW_AND_ADDED.inc()
         else:
             _LOGGER.info("Package %r in version %r was not added for tracking", package_name, package_version)
+            _METRIC_PACKAGES_NEW_JUST_DISCOVERED.inc()
```

Comment:
This is kind of confusing for me. We mark a new release in the graph database if we've already seen that package being used in software stacks, otherwise we just simply skip packages that are not used anywhere (for us). This metric should be always equal to 40 - _METRIC_PACKAGES_NEW_AND_ADDED (considering 40 to be number of last packages released on PyPI RSS feed).

Reply:
it should be 40-added packages?!